server/bin/pbench-index.py

```diff
@@ -102,6 +119,45 @@ def main(options, name):
          8 - Unable to load and process expected mapping files
          9 - Unable to update index templates in configured Elasticsearch
              instance
+
+        Signal Handlers used to establish different patterns for the four
+        behaviors:
+
+        1. Gracefully stop processing tar balls
+            - SIGQUIT
+            - The current tar ball is indexed until completion, but no other
+              tar balls are processed.
+            - Handler Behavior:
+                - Sets a flag that causes the code flow to break out of the
+                  for loop.
+                - Does not raise an exception.
+
+        2. Interrupt the current tar ball being indexed, and proceed to the
+           next one, if any
+            - SIGINT
+            - Handler Behavior:
+                - try/except/finally placed immediately around the es_index()
+                  call so that the signal handler will only be established for
+                  the duration of the call.
+                - Raises an exception caught by above try/except/finally.
+                - The finally clause would take down the signal handler.
+
+        3. Report the current state of indexing
+            - SIGHUP
+            - Current tar ball being processed
+            - How many remaining
+            - How many have been completed
+            - Errors encountered
+            - Handler Behavior:
+                - Kicks a special thread to log indexing state data
+                - No exception raised
```

Comment:
Since we're not actually doing this, we shouldn't be documenting it.

Reply:
@riya-17, can you file an issue to track the above instead?

Reply:
yeah sure